src/find_controls.cpp

```diff
@@ -194,6 +194,7 @@ const char* ToCString(const FindMode::Values mode)
     case FindMode::Values::Normal: return "NORMAL";
     case FindMode::Values::Fast: return "FAST";
     case FindMode::Values::Hybrid: return "HYBRID";
+    case FindMode::Values::OptimizedHybrid: return "OPTIMIZED_HYBRID";
```

Comment:
Why add a new mode, this looks like a heuristic which should just be part of the "Hybrid" mode. @atamazov comments ?

Reply:
This heuristic has [negative side effects](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/299#issuecomment-645698370) and [looks like a patch](https://github.com/ROCmSoftwarePlatform/MIOpen/issues/299#issuecomment-646140162) from the architectural POV. That is why I would like to keep "plain" Hybrid.